projects/grandine-windows-support.md

```diff
@@ -0,0 +1,70 @@
+# Grandine Windows Support
+
+To support building Grandine on Windows.
+
+## Motivation
+
+Grandine's developers mainly use Linux, so Grandine is tested only on Linux. However, there are many Windows and MacOS users that would benefit from better support. Furthermore, another Ethereum consensus client [Lighthouse](https://github.com/sigp/lighthouse) already supports Windows. 
+
+## Project description
+
+The goal of this project is to extend support for building Grandine, an Ethereum consensus client, on Windows. While Grandine has been primarily developed and tested on Linux, expanding its compatibility to Windows will make it more accessible to a broader range of developers and users.
+
+This project seeks to ensure that Windows users can build, run, and/or contribute to Grandine without platform-specific issues. With another Ethereum consensus client, Lighthouse, already supporting Windows, this project will make Grandine to catch up with other clients in terms of cross-OS support.
+
+## Specification
```

Comment:
The specification needs to include details of your planned work. For example::
- updating build system with windows specific packages
- updating documentation with new instructions.. etc